.github/workflows/add-model-like.yml

```diff
@@ -27,14 +27,27 @@ jobs:
         id: cache
         with:
           path: ~/venv/
-          key: v2-tests_model_like-${{ hashFiles('setup.py') }}
+          key: v3-tests_model_like-${{ hashFiles('setup.py') }}
 
       - name: Create virtual environment on cache miss
         if: steps.cache.outputs.cache-hit != 'true'
         run: |
           python -m venv ~/venv && . ~/venv/bin/activate
           pip install --upgrade pip!=21.3
-          pip install .[dev]
+          pip install -e .[dev]
+
+      - name: Check transformers location
+        # make `transformers` available as package (required since we use `-e` flag) and check it's indeed from the repo.
+        run: |
+          . ~/venv/bin/activate
+          python setup.py build install
```

Comment:
Usually when installing it with `-e`, you should do `python setup.py develop`, no?
If this works well here, fine by me!

Reply:
Change it to `develop` as it makes more sense. Thank you for the information!